src/_version_check.sh

```diff
@@ -140,11 +144,14 @@ function version_check() {
           # Set Contiguous updates to false here to ensure that since the app is on latest version, it still attempts to pull patch version updates
           contiguous_update=false
         fi
-        # Sort the upstream tags weve chosen as the upgrade path
-        IFS=$'\n' upgrade_path=($(sort -V <<<"${upstream_tags[*]}"))
-        # Reset IFS to default value
-        IFS=$' \t\n'
-
+        if "${upstream_tags[@]}" != ""; then
+                # Sort the upstream tags weve chosen as the upgrade path
+                IFS=$'\n' upgrade_path=($(sort -V <<<"${upstream_tags[*]}"))
+                # Reset IFS to default value
+                IFS=$' \t\n'
+        else
+                upgrade_path=("")
+        fi
```

Comment:
Same thing here; This checks if the array length is greater than 0 and then attempts to use it. Otherwise it sets it to an expected value of ""